resources/views/site/blocks/feature_block.blade.php

```diff
@@ -129,8 +129,8 @@
                             @elseif( $feature_type === 'videos')
                                 <em class="type f-tag">Video</em>
                             @endif
-                            @if ($item->title)
-                                <strong class="title f-list-3">{{ $item->title }}</strong>
+                            @if ($item->present()->title)
```

Comment:
Since we know this is a model attribute, the if condition doesn't need to go through the presenter. If `$item->title` has a value, then continue on and output the value using the presenter.

Reply:
updated